File: tests/perf/test-prop-write.py
def test():
	obj = { 'xxx1': 1, 'xxx2': 2, 'xxx3': 4, 'xxx4': 4, 'foo': 123 }
	i = 0
	while i < 1e7:
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		obj['foo'] = 234
		i += 1

test()

File: tests/perf/test-prop-write.lua
function test()
    local obj = { xxx1 = 1, xxx2 = 2, xxx3 = 3, xxx4 = 4, foo = 123 }
    for i=1,1e7 do
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
        obj.foo = 234
    end
end

test()
